pkg/postage/batch.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package postage
+
+import (
+	"encoding/binary"
+	"math/big"
+)
+
+// Batch represents a postage batch, a payment on the blockchain.
+type Batch struct {
+	ID    []byte   // batch ID
+	Value *big.Int // overall balance of the batch
+	Start uint64   // block number the batch was created
+	Owner []byte   // owner's ethereum address
+	Depth uint8    // batch depth, i.e., size = 2^{depth}
+}
+
+// MarshalBinary serialises a postage batch to a byte slice len 117.
```

Comment:
actually this is `93`, as seen in `L23`. maybe better to just remove the length from the comment. from `Unmarshal` too

Reply:
Fixed, here and in `Unmarshal`